platform/runtimes/runtime-specification.mdx

```diff
@@ -8,11 +8,31 @@ sidebarTitle: 'Runtime Specification'
 By default, all our runners have their timezone set to UTC, regardless of their location.
```

Comment:
I'm missing a short description of what a runner is, or is that clear from context?